packages/hash/frontend/src/pages/[account-slug]/types/entity-type/[entity-type-id]/entities.page.tsx

```diff
@@ -0,0 +1,167 @@
+import { faMagnifyingGlass } from "@fortawesome/free-solid-svg-icons";
+import { GridCellKind } from "@glideapps/glide-data-grid";
+import {
+  Chip,
+  FontAwesomeIcon,
+  IconButton,
+} from "@hashintel/hash-design-system";
+import { Box, Paper, Stack } from "@mui/material";
+import { useMemo, useState } from "react";
+import { faCircleQuestion } from "@fortawesome/free-regular-svg-icons";
+import { GlideGrid } from "../../../../../components/GlideGlid/glide-grid";
+import { GlideGridOverlayPortal } from "../../../../../components/GlideGlid/glide-grid-overlay-portal";
+import {
+  createHandleHeaderClicked,
+  sortRowData,
+  TableSort,
+} from "../../../../../components/GlideGlid/utils/sorting";
+import { useDrawHeader } from "../../../../../components/GlideGlid/utils/use-draw-header";
+import { SectionWrapper } from "../../../shared/section-wrapper";
+import { WhiteChip } from "../../../shared/white-chip";
+import { blankCell } from "../../../../../components/GlideGlid/utils";
+import { HomeIcon } from "../../../../../shared/icons/home-icon";
+import { EarthIcon } from "../../../../../shared/icons/earth-icon";
+import { renderValueIconCell } from "../value-icon-cell";
+import { useRouteNamespace } from "../use-route-namespace";
+import { useEntitiesTable } from "../use-entities-table";
+import { useEntityTypeEntities } from "../use-entity-type-entities";
+import { NextPageWithLayout } from "../../../../../shared/layout";
+import { getEntityTypeEditorLayout } from "../entity-type-header";
+
+const Page: NextPageWithLayout = () => {
+  const { entities, entityTypes, propertyTypes, subgraph } =
+    useEntityTypeEntities() ?? {};
+
+  const [showSearch, setShowSearch] = useState(false);
+  const [tableSort, setTableSort] = useState<TableSort<string>>({
+    key: "entity",
+    dir: "desc",
+  });
```

Comment:
Can the search and sort logic live in the table instead, or some component wrapping it?

It feels like we'll want to use those in other places and we don't want every user of the table having to implement their own version (unless there's some reason to)

Reply:
I think it can be done in the `GlideGrid` component, but I suggest doing this in a follow-up PR.

Reply:
@luisbettencourt For work to be done in separate/follow-up PRs, can you create sub-tasks and x-link them so we can track?

Reply:
Not sure if it should be a subtask, it's related to our implementation of the `GlideGrid` component and will also change the tables on the entity page. Here's the [task ](https://app.asana.com/0/1201095311341924/1203374488722348/f) _(internal)_ I created for this.